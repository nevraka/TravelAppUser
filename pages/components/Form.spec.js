import { search } from './Form';

test('search', () => {
  let results = search('Berli');
  expect(results).toMatchSnapshot();
  results = search('istanb');
  expect(results).toMatchSnapshot();
});
