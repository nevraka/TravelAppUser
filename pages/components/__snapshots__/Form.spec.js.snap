// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search 1`] = `
Array [
  Object {
    "city": "Uberlingen",
    "country": "Germany",
    "flag": "ğŸ‡©ğŸ‡ª",
  },
  Object {
    "city": "Berlin",
    "country": "Germany",
    "flag": "ğŸ‡©ğŸ‡ª",
  },
  Object {
    "city": "Berliste",
    "country": "Romania",
    "flag": "ğŸ‡·ğŸ‡´",
  },
  Object {
    "city": "Berlin",
    "country": "El Salvador",
    "flag": "ğŸ‡¸ğŸ‡»",
  },
  Object {
    "city": "Berlin",
    "country": "United States",
    "flag": "ğŸ‡ºğŸ‡¸",
  },
  Object {
    "city": "Berlin",
    "country": "United States",
    "flag": "ğŸ‡ºğŸ‡¸",
  },
  Object {
    "city": "Oberlin",
    "country": "United States",
    "flag": "ğŸ‡ºğŸ‡¸",
  },
  Object {
    "city": "New Berlin",
    "country": "United States",
    "flag": "ğŸ‡ºğŸ‡¸",
  },
]
`;

exports[`search 2`] = `
Array [
  Object {
    "city": "Istanbul",
    "country": "Turkey",
    "flag": "ğŸ‡¹ğŸ‡·",
  },
]
`;
